packages/connection/src/class-manager.php

```diff
@@ -2167,4 +2167,15 @@ public function sign_role( $role, $user_id = null ) {
 
 		return $role . ':' . hash_hmac( 'md5', "{$role}|{$user_id}", $token->secret );
 	}
+
+	/**
+	 * Returns the detailed connection error message for the given error code.
+	 *
+	 * @param string $error_code The error code.
+	 *
+	 * @return string The error message.
+	 */
+	public function get_error_message_from_code( $error_code ) {
+		return ( new Connection_Error_Handler() )->get_error_message_from_code( $error_code );
```

Comment:
This will need renaming too now, won't it? I tried to find where it was `use`d, but it's not, so it's going to be accessed as `\Automattic\Jetpack\Connection\Connection_Error_Handler`, which doesn't exist anymore.